ddtrace/internal/telemetry/writer.py

```diff
@@ -453,6 +454,21 @@ def _app_started_event(self):
                     "origin": "env_var",
                     "value": str(config._propagation_style_extract),
                 },
+                {
+                    "name": "ddtrace_bootstrapped",
```

Comment:
@mabdinur something @XG-xin and I were talking about for https://github.com/DataDog/dd-trace-py/pull/6148

we might want to try and store these values differently, e.g. store them globally or as a class attribute rather than writing statically into the payload like this. since we need to compare against previous values in order to track changes we don't want to have to look through the payload for the previous values.

just calling it out as something that could change on you.

Reply:
Ahh okay. Ya this isn't the best approach. I just wanted to close out the otel work and then refactor later. I can hold off for now.

Reply:
I'll let the two of you decide how to handle. I don't see why order should matter too much, as long as you coordinate.

Reply:
I'll reach out to @XG-xin. I would prefer to get this in as soon as possible.

Reply:
@mabdinur I update my PR to store the configuration queue in the class attribute here:
https://github.com/DataDog/dd-trace-py/pull/6148/files#diff-0cb9315daf5ed5208de657467cc10a6b7dda616a161c9b5b1f8cff83c95894deR392
Does this look good to you?